Dockerfile

```diff
@@ -1,30 +1,47 @@
-FROM openjdk:11-jre
+## Builder Image
+FROM alpine:latest as builder
 
 ARG DISTBALL
 
-ENV ZB_HOME=/usr/local/zeebe \
+ENV TMP_ARCHIVE=/tmp/zeebe.tar.gz \
+    TMP_DIR=/tmp/zeebe \
+    TINI_VERSION=v0.19.0
+
+COPY ${DISTBALL} ${TMP_ARCHIVE}
+
+RUN mkdir -p ${TMP_DIR} && \
+    tar xfvz ${TMP_ARCHIVE} --strip 1 -C ${TMP_DIR} && \
+    # already create volume dir to later have correct rights
+    mkdir ${TMP_DIR}/data
+
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini ${TMP_DIR}/bin/tini
+COPY docker/utils/startup.sh ${TMP_DIR}/bin/startup.sh
+RUN chmod +x -R ${TMP_DIR}/bin/
+
+## Final Image
+FROM openjdk:11-jre
```

Comment:
If we're switching the pipeline any way, might as well go for `openjdk:11-jre-slim`? Then you don't need to purge apt repos and so on

Reply:
I will give it a try